.ci/magician/github/set.go

```diff
@@ -78,39 +78,46 @@ func (c *Client) UpdateComment(prNumber, comment string, id int) error {
 	return nil
 }
 
-func (gh *Client) RequestPullRequestReviewers(prNumber string, reviewers []string) error {
-	url := fmt.Sprintf("https://api.github.com/repos/GoogleCloudPlatform/magic-modules/pulls/%s/requested_reviewers", prNumber)
+// RequestPullRequestReviewers adds reviewers to a pull request
+func (c *Client) RequestPullRequestReviewers(prNumber string, reviewers []string) error {
+	num, err := strconv.Atoi(prNumber)
+	if err != nil {
+		return err
+	}
 
-	body := map[string][]string{
-		"reviewers":      reviewers,
-		"team_reviewers": {},
+	// Create the reviewers request
+	reviewersRequest := gh.ReviewersRequest{
+		Reviewers:     reviewers,
+		TeamReviewers: nil,
 	}
 
-	err := utils.RequestCallWithRetry(url, "POST", gh.token, nil, body)
+	_, _, err = c.gh.PullRequests.RequestReviewers(c.ctx, defaultOwner, defaultRepo, num, reviewersRequest)
```

Comment:
It looks like this line is causing a failure because it doesn't match the schema GitHub expects:

```
422 Invalid request.

No subschema in "anyOf" matched.
"reviewers" wasn't supplied.
"team_reviewers" wasn't supplied.
```


I think previously we were getting around this by accident because we were sending an empty object for team_reviewers - now we probably need to make this function a no-op if the list of reviewers is empty. (And we don't need to set TeamReviewers at all.)